service-deployment/src/rest-server/dockerfile.template

```diff
@@ -17,42 +17,29 @@
 
 FROM node:boron
 
-ENV HADOOP_VERSION={{ clusterconfig['clusterinfo']['hadoopinfo']['hadoopversion'] }}
+RUN echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
 
-RUN echo "deb http://http.debian.net/debian jessie-backports main" > \
-    /etc/apt/sources.list.d/jessie-backports.list && \
-    apt-get update && \
-    apt-get install -y --no-install-recommends -t jessie-backports \
-    openjdk-8-jre-headless ca-certificates-java dos2unix && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+RUN apt-get update
 
-COPY copied_file/hadoop/hadoop-{{ clusterconfig['clusterinfo']['hadoopinfo']['hadoopversion'] }}.tar.gz .
+RUN apt-get install -y --no-install-recommends -t jessie-backports dos2unix
 
-RUN tar -xzf hadoop-{{ clusterconfig['clusterinfo']['hadoopinfo']['hadoopversion'] }}.tar.gz -C /usr/local && \
-    mv /usr/local/hadoop-${HADOOP_VERSION} /usr/local/hadoop
-
-ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64 \
-    HADOOP_INSTALL=/usr/local/hadoop \
-    HADOOP_HDFS_HOME=${HADOOP_INSTALL} \
-    HADOOP_COMMON_LIB_NATIVE_DIR=${HADOOP_INSTALL}/lib/native \
-    HADOOP_OPTS="-Djava.library.path=${HADOOP_INSTALL}/lib/native"
-
-ENV PATH=$PATH:${HADOOP_INSTALL}/bin
-ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${JAVA_HOME}/jre/lib/amd64/server
+RUN apt-get clean
 
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
run those in a single layer

Reply:
Could you explain the benefits of doing so?

Reply:
please refer to [docker docs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers)